controller/volume_controller.go

```diff
@@ -1068,7 +1074,7 @@ func (vc *VolumeController) createReplica(v *longhorn.Volume, e *longhorn.Engine
 			BaseImage:  v.Spec.BaseImage,
 		},
 	}
-	if v.Spec.FromBackup != "" {
+	if v.Spec.FromBackup != "" && v.Spec.Standby {
```

Comment:
We need to update the standby volume to use the same mechanism.

Reply:
Standby volume uses incremental backup and it is already being checked every 5 seconds by engine monitor. I was thinking of having the status check in the engine monitor code itself, something like `IncrementalBackupRestoreStatus` similar to `BackupRestoreStatus`

Reply:
You should be able to use the same mechanism, since it's still one restore at a time. We only need the latest restoration progress anyway.